### PR TITLE
BaSP-MR-112: implement redux subscription

### PR DESCRIPTION
**Pull request by GaloDurante**
I worked adding Redux on the subscriptions resource and implement a Spinner when the page is loading

### CODE REVIEW

**Comment by LucianoStradiot** [comment]
[Implement Redux on the Subscriptions entity](https://trello.com/c/3gtwy2aB/112-implement-redux-on-the-subscriptions-entity)

File: src/Components/Subscriptions/index.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+import { getSubscriptions, deleteSubscription } from '../../Redux/Subscriptions/thunks';
 import style from './subscriptions.module.css';
 import Modal from '../Shared/Modal';
 import Button from '../Shared/Button';
+import Spinner from '../Shared/Spinner';
 import { Link } from 'react-router-dom';
 
 function Subscriptions() {
-  const [subscriptions, setSubscriptions] = useState([]);
+  //const [subscriptions, setSubscriptions] = useState([]);
```
Comment (Tilajc) [inline src/Components/Subscriptions/index.js]: You can delete this
```suggestion
  
```

**Review by Tilajc — CHANGES_REQUESTED** [review]
Pretty good job, just delete the comment!

**Review by Tilajc — APPROVED** [review]
gj

**Comment by Katairala** [comment]
Tested locally on https://joaco-megarocket-app.vercel.app/
Test Results: Passed 🌟

It looks great Galo!

**Pull request merged**